_includes/update23/requirements-media-update.md

```diff
@@ -1,6 +1,6 @@
 First we should make sure that the latest drush version is installed.
 ```
-composer require drush/drush:~9.7.0 --no-update
+composer require drush/drush:~10.0.0 --no-update
```

Comment:
This will keep us on 10.0.*, better use  `composer require drush/drush:~10.0 --no-update`